mettle/worker/redis.go

```diff
@@ -38,17 +36,19 @@ var redisLatency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
 // All usage of Redis is best-effort only.
 // If readRedis is set, all reads will happen on this client. If not, everything
 // will go to the primary client.
-func newRedisClient(client elan.Client, primaryRedis, readRedis *redis.Client) elan.Client {
+func newRedisClient(client elan.Client, primaryRedis, readRedis *redis.Client, maxSize int64) elan.Client {
 	// This is a safeguard in case the caller does not pass readRedis.
 	if readRedis == nil {
 		readRedis = primaryRedis
 	}
+	if maxSize <= 0 {
+		maxSize = 200 * 1012 // 200 Kelly-Bootle standard units
```

Comment:
We have this set in two places but we will always want it to be the same (probably). Should we make this a exportable const in the redis library so we have maxSize = redis.DefaultMaxSize

Reply:
Ah yeah, good idea